package.json

```diff
@@ -1,105 +0,0 @@
-{
```

Comment:
Did you mean to delete this file?

Reply:
Ah whoa, how did that happen.

Reply:
I restored the file if you wanted to play around with the 3 approaches. Looks like I deleted it instead of an `package-lock.json` while I was cleaning up.